doc/source/whatsnew/v1.3.0.rst

```diff
@@ -601,6 +601,7 @@ ExtensionArray
 
 - Bug in :meth:`DataFrame.where` when ``other`` is a :class:`Series` with :class:`ExtensionArray` dtype (:issue:`38729`)
 - Fixed bug where :meth:`Series.idxmax`, :meth:`Series.idxmin` and ``argmax/min`` fail when the underlying data is :class:`ExtensionArray` (:issue:`32749`, :issue:`33719`, :issue:`36566`)
+- Bug where some properties of subclasses of :class:``PandasExtensionDtype`` where improperly cached :issue:`40329`
```

Comment:
where -> were

parens around the issue tag

single-ticks around PandasExtensionDtype